lms/djangoapps/support/tests/test_views.py

```diff
@@ -1189,3 +1189,124 @@ def test_fbe_disabled_response(self):
         )
         data = json.loads(response.content.decode('utf-8'))
         assert data == {}
+
+
+class LinkProgramEnrollmentSupportAPIViewTests(SupportViewTestCase):
+    """
+    Tests for the link_program_enrollments support view.
+    """
+
+    def setUp(self):
+        """Make the user support staff. """
+        super().setUp()
+        self.url = reverse("support:link_program_enrollments_details")
+        SupportStaffRole().add_users(self.user)
+        self.program_uuid = str(uuid4())
+        self.text = '0001,user-0001\n0002,user-02'
+
+    def test_invalid_uuid(self):
+        response = self.client.post(self.url, data={
+            'program_uuid': 'notauuid',
+            'text': self.text,
+        })
+        msg = "Supplied program UUID 'notauuid' is not a valid UUID."
+        data = json.loads(response.content.decode('utf-8'))
+        assert data['errors'] == [msg]
+
+
+    def test_missing_parameter(self):
+        data = [
+            ('program_uuid', ''),
+            ('', 'text'),
+            ('', ''),
+        ]
```

Comment:
we should use ddt for data-driven tests instead of having for loops in the test.

Reply:
Updated to ddt